classes/book.rb

```diff
@@ -0,0 +1,15 @@
+require './classes/item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(publisher, publish_date, cover_state)
+    super(publish_date)
+    @publisher = publisher
+    @cover_state = cover_state
+  end
+
+  def can_be_archived?
+    @archived || cover_state == 'bad'
+  end
```

Comment:
- [ ] the `can_be_archived?` method should return true if parent's method  returns true OR if cover_state equals to "bad". in this case, you're checking the `@archived` instance variable, this should be `super` like in `Game` class and `Music_album` class

Reply:
Addressed this!